content/intro-to-storybook/ember/en/test.md

```diff
@@ -0,0 +1,129 @@
+- **Unit tests** with Qunit verify that the output of a component remains the same given a fixed input. They’re great for testing the functional qualities of a component.
+
+## “But does it look right?”
+
+Unfortunately, the aforementioned testing methods alone aren’t enough to prevent UI bugs. UIs are tricky to test because design is subjective and nuanced. Visual tests are too manual, snapshot tests trigger too many false positives when used for UI, and pixel-level unit tests are poor value. A complete Storybook testing strategy also includes visual regression tests.
+
+## Visual regression testing for Storybook
+
+Visual regression tests are designed to catch changes in appearance. They work by capturing screenshots of every story and comparing them commit-to-commit to surface changes. This is perfect for verifying graphical elements like layout, color, size, and contrast.
+
+<video autoPlay muted playsInline loop style="width:480px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/visual-regression-testing.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Storybook is a fantastic tool for visual regression testing because every story is essentially a test specification. Each time we write or update a story we get a spec for free!
+
+There are a number of tools for visual regression testing. For professional teams we recommend [**Chromatic**](https://www.chromaticqa.com/), an addon made by Storybook maintainers that runs tests in the cloud.
+
+## Setup visual regression testing
+
+Chromatic is a hassle-free Storybook addon for visual regression testing and review in the cloud. Since it’s a paid service (with a free trial), it may not be for everyone. However, Chromatic is an instructive example of a production visual testing workflow that we'll try out for free. Let’s have a look.
+
+### Bring git up to date
+
+Ember cli has created a git repo for your project; let's check in the changes we have made:
+
+```bash
+$ git add -A
+```
+
+Now commit the files.
+
+```bash
+$ git commit -m "taskbox UI"
+```
+
+### Get Chromatic
+
+Add the package as a dependency.
+
+```bash
+npm install -D storybook-chromatic
+```
+
+One fantastic thing about this addon is that it will use Git history to keep track of your UI components.
+
+Then [login to Chromatic](https://www.chromaticqa.com/start) with your GitHub account (Chromatic only asks for lightweight permissions). Create a project with name "taskbox" and copy your unique `app-code`.
+
+<video autoPlay muted playsInline loop style="width:520px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/chromatic-setup-learnstorybook.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Run the test command in the command line to setup visual regression tests for Storybook. Don't forget to add your unique app code in place of `<app-code>`.
+
+```bash
+npx chromatic --app-code=<app-code>
+```
+
+<div class="aside">
```

Comment:
Let's remove this aside, because it's no longer necessary with the new Chromatic CLI :)

Reply:
It was fixed with the latest commit. Going to mark it as resolved if you don't mind.